include/couch_replicator_api_wrap.hrl

```diff
@@ -24,7 +24,8 @@
     retries = 10,
     wait = 250,         % milliseconds
     httpc_pool = nil,
-    http_connections
+    http_connections,
+    backoff = 250
```

Comment:
start lower? we're expecting rate limits of 10's or 100's of requests per second. If this were runtime configurable I would worry less. Bidding starts at 25.